README.md

```diff
@@ -18,12 +18,31 @@ $ conda config --add channels conda-forge
 $ conda install cookiecutter
 ```
 
+ - dvc >= 0.93
+
 
 ### To start a new project, run:
 ```
 cookiecutter https://github.com/analysiscenter/TaskCookieCutter
 ```
 
+You will be asked for these fields:
+| Field                 | Default                                                       |  Description    |
+|-----------------------|---------------------------------------------------------------|-----------------|
+| project_name          | `Template`                                                    | Verbose project name, used in headings (docs, readme, etc).  |
+| repo_name             | `project_name` lowercased with spaces replaced by underscores | Repository name on GitHub (and project's root directory name). |
+| include_library       | `no`                                                          | Chose a library to attach as a git submodule. `no` means that no library will be attached. |
+| git_config_user_name  | `do_not_set`                                                  | Specify local user name for git. Default is to skip this. |
+| git_config_user_email | `do_not_set`                                                  | Specify local user email for git. Default is to skip this. |
+| init_DVC              | `yes`                                                         | Select whether you want to initialize DVC in the project. |
+| dvc_cache_dir_mode    | `pass` if you selected not to initialize DVC, else `infer`    | Select cache directory for DVC. If `pass`, nothing is done; if `infer`, path is selected as `/data/<repo_name>/.dvc/cache` depending on `include_library`, if `custom`, value from `custom_dvc_cache_dir` is used |
+| custom_dvc_cache_dir  | `pass`                                                        | Provide a path to DVC cache directory, if you selected `custom` as `dvc_cache_dir_mode`. |
+| lib_dvc_cache_dir_map | `default`                                                     | Just press `Enter` :) This sets some utility values. |
```

Comment:
Stupid question. Do we really need this?

Reply:
I'm afraid I can't fully understand your question. 

If your question is about existance of this table, the answer is: There were questions about meaning of different settings in the template's menu, so I thought it might be usefull to add a full description in README

Reply:
I mean the last parameter that we can define only as `default`. I don't understand why we should define it in this way by pressing `Enter`.

Reply:
This setting is used both in pre and post generation hooks and adding it to the json file seemed  to me the less ugly way to have it in both scripts, since hooks scripts in cookiecutter can't do local imports by now (though there is some very recent work about it https://github.com/cookiecutter/cookiecutter/pull/980)

The other option would be to define `lib_dvc_cache_dir_map` both in pre_gen_project.py and post_gen_project.py